File: sop-sdk/sdk-csharp/SDKCSharp/Response/DemoFileUploadResponse.cs
using System;
using System.Collections;
using System.Collections.Generic;
using Newtonsoft.Json;

namespace SDKCSharp.Response
{
    public class DemoFileUploadResponse : BaseResponse
    {
        [JsonProperty("files")]
        private List<FileMeta> files = new List<FileMeta>();

        public List<FileMeta> Files { get => files; set => files = value; }

        public class FileMeta
        {

            public FileMeta(String filename, long size, String content)
            {
                this.Filename = filename;
                this.Size = size;
                this.Content = content;
            }

            public FileMeta()
            {
            }

            [JsonProperty("filename")]
            public String Filename { get; set; }

            [JsonProperty("size")]
            public long Size { get; set; }

            [JsonProperty("content")]
            public String Content { get; set; }
        }
    }
}

File: sop-sdk/sdk-java/src/main/java/com/gitee/sop/sdk/response/DemoFileUploadResponse.java
package com.gitee.sop.sdk.response;

import lombok.Data;
import lombok.EqualsAndHashCode;

import java.util.ArrayList;
import java.util.List;

/**
 * @author tanghc
 */
@Data
@EqualsAndHashCode(callSuper = false)
public class DemoFileUploadResponse extends BaseResponse {

    private List<FileMeta> files = new ArrayList();

    @Data
    public static class FileMeta {

        public FileMeta(String filename, long size, String content) {
            this.filename = filename;
            this.size = size;
            this.content = content;
        }

        public FileMeta() {
        }

        private String filename;
        private long size;
        private String content;
    }
}
